Remove warning from Plugin Generator

yuki24 opened the pull request:
This removes the following warning:

```
rails/railties/lib/rails/generators/rails/plugin/plugin_generator.rb:321: warning: shadowing outer local variable - content
```

vipulnsward commented:
LGTM

rafaelfranca commented on railties/lib/rails/generators/rails/plugin/plugin_generator.rb:
```diff
@@ -319,10 +319,10 @@ def modules
       def wrap_in_modules(content)
         content = "#{content}".strip.gsub(/\W$\n/, '')
-        modules.reverse.inject(content) do |content, mod|
+        modules.reverse.inject(content) do |kontent, mod|
```
I think it is better to rename the outside variable to `unwrapped_code`

kaspth replied:
I'll second that. `kontent` sounds like a typo (even though it mimics the `klass` convention).

yuki24 replied:
`unwrapped_code` sounds better! i'll change it.

yuki24 committed:
Remove warning from Plugin Generator

This removes the following warning:

  rails/railties/lib/rails/generators/rails/plugin/plugin_generator.rb:321: warning: shadowing outer local variable - content

yuki24 commented:
updated commit. Thanks @vipulnsward @rafaelfranca for reviewing!

senny commented:
@yuki24 thank you.

rafaelfranca commented:
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
